### DISCUSSION TITLE
Next Image can't load local images from api in localhost:8000/storage/file.jpg

### DISCUSSION THREAD

**Discussion by kayooliveira** [vercel/next.js#58354] [Help] [answered]
### Summary

Hello guys, i'm trying to load images from localhost:8000 in my next aplication, however when the app loads the Image component from 'next/image' the image not work and paste an error on the console: 

![image](https://github.com/vercel/next.js/assets/35928107/8e1707bd-d14a-4bff-9f10-50374f9d767e)

The Image URL From Database Column: http://localhost:8000/storage/products/adesivo-perfurado-m655103c7c3b87.jpg

My NextConfig
![image](https://github.com/vercel/next.js/assets/35928107/6dca9b27-9b8e-43d6-b345-a6ad5c0ba83c)

other images from other urls works perfectly, but the image with localhost:8000 no
![image](https://github.com/vercel/next.js/assets/35928107/53f1a3b7-64cc-4b0a-b379-bbcf57c8b192)

### Additional information

```js
Operating System: 
    Platform: Windows
Binaries: 
    Node: 18.18.2
    Next: 14.0.1
```

**Comment by john-prutton** [comment]
Hi. My first thought is that it is assuming the protocol to be HTTPS. Why not try specifying the protocol directly:

```ts
images: {
  remotePatterns: [
    {
      protocol: 'http',
      hostname: 'localhost:8000',
    },
  ]
}
  ```

**Reply by kayooliveira** [reply, marked as answer]
I changed it from localhost to 127.0.0.1 and it worked.
